Sources/SwiftLayout/Printing/SwiftLayoutPrinter.swift

```diff
@@ -8,6 +8,22 @@
 import Foundation
 
 public struct SwiftLayoutPrinter: CustomStringConvertible {
+    
+    public struct PrintOptions: OptionSet {
+        public let rawValue: Int
+        
+        public init(rawValue: Int) {
+            self.rawValue = rawValue
+        }
+        
+        /// print system constraints
+        public static let withSystemConstraints: PrintOptions = .init(rawValue: 1)
+        /// print view only have accessibility identifier
+        public static let onlyIdentifier: PrintOptions = .init(rawValue: 1 << 1)
+        /// print with view config
+        public static let withViewConfig: PrintOptions = .init(rawValue: 1 << 2)
```

Comment:
Is this option currently available?
It seems that this option is not yet available

ps. In fact, it is a very necessary option when preparing for performance tests. 
I'm very surprised that the options is included in this pr.

Reply:
hahaha it works(a little bit) in next PR.